sklearn/linear_model/omp.py

```diff
@@ -539,7 +539,7 @@ def orthogonal_mp_gram(Gram, Xy, n_nonzero_coefs=None, tol=None,
         return np.squeeze(coef)
 
 
-class OrthogonalMatchingPursuit(LinearModel, RegressorMixin, MultiOutputMixin):
+class OrthogonalMatchingPursuit(MultiOutputMixin, RegressorMixin, LinearModel):
```

Comment:
kinda same here, confused.

Reply:
Generally: the more specialized, the more to the left. And all the mixins are more specialized than all the estimators because that's the way mixins work. I argue in https://github.com/scikit-learn/scikit-learn/issues/14044#issuecomment-520509718 that I think this is silly and confusing.